src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  box-sizing: border-box;
```

Comment:
you have already added 'box-sizing: border-box;' for the `*{}`, so it's redundant here